### PR TITLE
fix: golangci failing for doc/verifiable

### LINKED ISSUE #1003: golangci failing for doc/verifiable
level=warning msg="[runner] Can't run linter goanalysis_metalinter: bodyclose: failed prerequisites: buildssa@github.com/hyperledger/aries-framework-go/pkg/doc/verifiable [github.com/hyperledger/aries-framework-go/pkg/doc/verifiable.test]"

level=warning msg="[runner] Can't run linter unused: buildssa: analysis skipped: errors in package: [/opt/workspace/pkg/doc/verifiable/credential.go:17:2: could not import github.com/VictoriaMetrics/fastcache (/go/pkg/mod/github.com/!victoria!metrics/fastcache@v1.5.4/malloc_mmap.go:24:24: Mmap not declared by package syscall) -: could not load export data: no export data for \"github.com/hyperledger/aries-framework-go/pkg/doc/verifiable\"]"

### PR DESCRIPTION
**Pull request by fqutishat**
closes #1003

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/doc/verifiable/credential.go [hunk 1]
```diff
@@ -223,16 +222,25 @@ type SchemaCache interface {
...
-	cache      *fastcache.Cache
+	cache      cache
 	expiration time.Duration
 }
 
 // NewExpirableSchemaCache creates new instance of ExpirableSchemaCache.
 func NewExpirableSchemaCache(size int, expiration time.Duration) *ExpirableSchemaCache {
 	return &ExpirableSchemaCache{
-		cache:      fastcache.New(size),
+		cache:      newCache(size),
```
Comment (troyronda) [inline pkg/doc/verifiable/credential.go]: It almost seems like `NewExpirableSchemaCache` should have an argument of the cache interface type.

Comment (fqutishat) [reply pkg/doc/verifiable/credential.go]: I moved NewExpirableSchemaCache

File: pkg/doc/verifiable/cache_js_wasm.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import "time"
+
+// NewExpirableSchemaCache creates new instance of ExpirableSchemaCache.
+func NewExpirableSchemaCache(size int, expiration time.Duration) *ExpirableSchemaCache {
```
Comment (troyronda) [inline pkg/doc/verifiable/cache_js_wasm.go]: I guess this function represents the default Cache implementation.

Comment (fqutishat) [reply pkg/doc/verifiable/cache_js_wasm.go]: yes

**Pull request merged**